### PR TITLE
docs(performance-detector-thresholds): Updated lowered default values…

### PR DESCRIPTION
**Pull request by Abdkhan14**
- We are GA'iNG the[ detector thresholds configuration ](https://www.notion.so/sentry/Detector-Threshold-Configuration-f8cb07e7cceb42388cedb09ea05fc116) project with lowered thresholds for performance issues.
- Updated detection criteria to respect the new values.

### CODE REVIEW

File: src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx [hunk 1]
```diff
@@ -15,8 +15,9 @@ Sentry detects N+1 API Calls in frontend and mobile applications. The detector i
 The N+1 API Call detector looks for groups of simultaneous network calls to the same URL. It uses the following detection criteria:
 
 - Applies to `GET` requests only
-- Minimum of 10 network calls must be happening simultaneously
-- Network calls must be a minimum of 50ms in duration and within 5ms of each other
+- Minimum of `10` network calls must be happening simultaneously
+- Total duration of involved spans must exceed `300ms`.
```
Comment (mjq-sentry) [inline src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx:19]: Comparing this doc to the other two, it looks like this one might be missing the note that you can configure this threshold on the Project Settings > Performance page?

**Review by mjq-sentry — APPROVED** [review]
Approving, but might want to tack on the missing sentence to this PR if it is indeed missing (see comment) :+1:

File: src/docs/product/issues/issue-details/performance-issues/uncompressed-asset/index.mdx [hunk 2]
```diff
@@ -14,7 +14,7 @@ The detector for this performance issue looks for specific asset (resource) span
 
 - The span has to be one of either: `resource.link` or `resource.script` spans, which in practice means it's a `.css` or `.js` file.
 - It has to be over `512kb` in size, (to filter out smaller files that are less likely to benefit from being compressed).
-- The span duration has to be over `500ms`, (to filter out faster transfers or assets served in internal networks).
+- The span duration has to be over `300ms`, (to filter out faster transfers or assets served in internal networks).
```
Comment (vivianyentran) [inline src/docs/product/issues/issue-details/performance-issues/uncompressed-asset/index.mdx:17]:
```suggestion
- The span duration has to be over `300ms` (to filter out faster transfers or assets served in internal networks).
```

File: src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx (hunk 1 above)
Comment (vivianyentran) [reply src/docs/product/issues/issue-details/performance-issues/n-one-api-calls.mdx:19]: I think you can configure N+1 DB queries but not N+1 API calls, but would be good to confirm.

**Review by vivianyentran — APPROVED** [review]
Nice, thanks for updating the docs!

**Pull request merged**